templates/publisher/listing.html

```diff
@@ -14,7 +14,7 @@
     <input type="hidden" name="snap_id" value="{{ snap_id }}" />
 
     <div class="snapcraft-p-sticky" id="store-listing-notification">
-      <div class="row">
+      <div class="row u-no-margin--bottom">
```

Comment:
Rows don't have margin in Vanilla.
It seems that we need to reset it because we set margin with custom `.market-form .row`, which is bit weird.

Not sure if we need to fix it now, but seems like layout of this page could use updating to remove such oddities.

Reply:
Yeah I think we should do a pass on the listings page after these vanilla updates to try and remove some of these silly overrides. At the end of the day it's a form, so should just work like the rest of the vanilla forms